Whitelisting.py

```diff
@@ -386,11 +395,15 @@ def check(self, pkg):
                 digest_matches, results = audit.compareDigests(pkg)
 
                 if digest_matches:
+                    for r in results:
+                        already_tested.add(r.path())
```

Comment:
nit: with a slight change to `compareDigests()` (returning the paths) this can be reduced to `already_tested |= results`

Reply:
I'm not quite sure what you mean. I need the results data structure the way it is, so you're suggesting to return a third argument with redundant information?

Reply:
No, I just didn't see that the results are ultimately used in `_printVerificationResults`. Leave things as-is then.